bin/kano-init-flow

```diff
@@ -17,83 +17,43 @@
 try:
     import os
     import sys
-    from gi.repository import Gtk, Gdk
+    import gettext
+    from gi.repository import Gtk
 
     if __name__ == '__main__' and __package__ is None:
         DIR_PATH = os.path.abspath(
             os.path.join(os.path.dirname(__file__), '..'))
         if DIR_PATH != '/usr':
             sys.path.insert(1, DIR_PATH)
+            LOCALE_PATH = os.path.join(DIR_PATH, 'locale')
+        else:
+            LOCALE_PATH = None
+
+    gettext.install('kano-init-flow', LOCALE_PATH, unicode=1)
 
-    from kano.gtk3.apply_styles import apply_common_to_screen
-    from kano.gtk3.application_window import ApplicationWindow
     from kano.utils import enforce_root
+
+    from kano_init_flow.main_window import MainWindow
     from kano_init_flow.first_screen import FirstScreen
     from kano_init_flow.audio_screen import AudioScreen
-    from kano_init_flow.reboot_screen import RebootScreen
-    from kano_init_flow.paths import CSS_DIR
-
-    enforce_root('Please run it as root / sudo user!')
-
-    # Window class
-    class MainWindow(ApplicationWindow):
-
-        def __init__(self, first_screen, reboot=False):
-            apply_common_to_screen()
-            specific_css = Gtk.CssProvider()
-            css_path = os.path.join(CSS_DIR, "kano-init-flow.css")
-            specific_css.load_from_path(css_path)
-            screen = Gdk.Screen.get_default()
-            styleContext = Gtk.StyleContext()
-            styleContext.add_provider_for_screen(screen, specific_css, Gtk.STYLE_PROVIDER_PRIORITY_USER)
-            # We need this for backwards compatibility
-            self.need_reboot = reboot
-            # Create main window
-            ApplicationWindow.__init__(self, title="Kano", width=2, height=2)
-            # This needs to be here to get the resizing correct for the screens
-            self.show_all()
-            # This is to get position right when the window resizes
-            self.set_position(Gtk.WindowPosition.CENTER_ALWAYS)
-            # Start on first_screen
-            first_screen(self)
-
-        def clear_win(self):
-            self.remove_main_widget()
-
-        # hacky fix - reset height allocation for the window
-        def reset_allocation(self):
-
-            allocation = self.get_allocation()
-            allocation.width = 1
-            allocation.height = 1
-            self.size_allocate(allocation)
-
-        # hacky fix - shrink height of window
-        def shrink(self):
-            self.resize(590, 100)
 
-        def exit_flow(self):
-            if self.need_reboot:
-                self.clear_win()
-                RebootScreen(self)
-            else:
-                # Init flow completed
-                sys.exit(0)
+    enforce_root(_('Please run it as root / sudo user!'))
```

Comment:
I've just checked [`enforce_root`](https://github.com/KanoComputing/kano-toolset/blob/d12e1b60c6c0e852f41932561f3043bf311a961d/kano/utils.py#L351-L354) in toolset. It does

```
sys.stderr.write(msg + "\n")
```

which, from a quick test, seems fine with unicode.